lib/routes/token.js

```diff
@@ -249,6 +249,18 @@ function confirmRefreshToken(params) {
       throw AppError.invalidScopes();
     }
     tokObj.scope = params.scope;
+
+    var now = new Date();
+    var lastUsedAt = tokObj.lastUsedAt;
+    var minimum_LastUsedAt_Delay_MS = 60000; // one minute. TODO: make it 24h
+
+    if ((lastUsedAt - now) > minimum_LastUsedAt_Delay_MS){
```

Comment:
Will `lastUsedAt` always be less than `now`, making `lastUsedAt - now` negative and causing this check to always fail?